### PR TITLE
Make some small fixes to Triage

### PR DESCRIPTION
**Pull request by michaelkolber**
ref: #18726

As I come across small mistakes, I'll add them to this PR

### CODE REVIEW

File: triage/summarize/files.go [hunk 1]
```diff
@@ -145,12 +145,14 @@ string, no message is printed.
 */
 func memoizeResults(filepath string, message string, v interface{}) {
```
Comment (MushuEE) [inline triage/summarize/files.go:146]: nit: (filepath, message string, v interface{})

File: triage/summarize/files.go [hunk 2]
```diff
@@ -143,14 +143,16 @@ prints a warning if the results could not be memoized.
...
-		logInfo("Done " + message)
+	if err != nil {
+		logWarning("Could not memoize results to '%s': %s", filepath, err)
 		return
 	}
 
-	logWarning("Could not memoize results to '%s': %s", filepath, err)
+	if message != "" {
+		logInfo("Done " + message)
+	}
```
Comment (michaelkolber) [inline triage/summarize/files.go:155]: This change will eventually need to use klog, so it should only be merged after #18858

**Comment by michaelkolber** [comment]
Use klog for the logging fix

**Pull request merged**